Source/Core/Common/MsgHandler.cpp

```diff
@@ -100,7 +100,16 @@ bool DefaultMsgHandler(const char* caption, const char* text, bool yes_no, int S
 	return IDYES == MessageBox(0, UTF8ToTStr(text).c_str(), UTF8ToTStr(caption).c_str(), STYLE | (yes_no ? MB_YESNO : MB_OK));
 #else
 	printf("%s\n", text);
-	return true;
+	
+	if (yes_no)
```

Comment:
I could also just return false, as anything not asking a question won't care what the result is.